mne/utils.py

```diff
@@ -2015,7 +2016,10 @@ def _get_stim_channel(stim_channel, info, raise_error=True):
     if ch_count > 0:
         return stim_channel
 
-    if 'STI 014' in info['ch_names']:
+    if 'STI101' in info['ch_names']:  # newer Elekta systems
+        return ['STI101']
```

Comment:
At least in the MGH system (and probably all older systems?) all channel names had this sort of space like "MEG 2443", and now they do not